airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -189,21 +225,11 @@
   documentationUrl: https://docs.airbyte.io/integrations/destinations/snowflake
   icon: snowflake.svg
   resourceRequirements:
-    # this is our first example of setting these requirements. they are guesses, not data driven.
-    # setting default cpu higher, because we have found that check and discover can be cpu constrained for dbs.
-    default:
-      cpu_limit: "1.0"
-      cpu_request: "1.0"
-      memory_limit: "300Mi"
-      memory_request: "300Mi"
     jobSpecific:
-      # sync jobs are generally IO and memory bound and not cpu.
       - jobType: sync
         resourceRequirements:
-          cpu_limit: "0.5"
-          cpu_request: "0.5"
```

Comment:
I removed the cpu_limit and cpu_request requirements because we are going to set the default CPU requirements for all jobs to 0.5, and if we decide to change that in the future then having this requirement here will prevent that change from affecting snowflake connectors.